content/en/docs/contributing/policy.md

```diff
@@ -0,0 +1,61 @@
+
+cert-manager currently works around this by shipping the CRDs in the templates. 
+
+## Helm Subchart capabilities
+
+cert-manager should not be used as a sub-chart in Helm deployments.
+Helm deployments are namespaced by design and do not support a dependency model that would support cert-manager being operated on a cluster scope nor preventing cert-manager from being installed twice (with conflicting versions) or being upgraded independently from the application.
+
+The cert-manager installation creates cluster scoped resources like admission webhooks and custom resource definitions. cert-manager should be seen as part of your cluster and should be treated as such for being installed. A comparison to other Kubernetes components would be a LoadBalancer controller or a PV provisioner
+
+## Secret injection or copying
+
+cert-manager deals with very sensitive information (all TLS certificates for your services) and has cluster-level access to secret resources, therefore when designing features we need to consider all ways these can be abused to escalate privilege.
+Secret data is meant to be securely stored in the secret resources and have narrow scoped access privileges for unauthorized users. Therefore we will not allow any functionality that allows this data to be copied/injected into any non secret resource. 
+
+### cainjector
+
+The cainjector component is a special exception to this rule. This component is able to inject the `ca.crt` file into predefined fields on `ValidatingWebhookConfiguration`, `MutatingWebhookConfiguration`, and `CustomResourceDefinition` resources from Certificate resources.
+These 3 components are already scoped only for privileged users, and will already give you cluster scoped access to resources. 
+
+If you’re designing a resource that needs a CA Certificate or TLS key pair it is strongly recommended to use a reference to a secret instead of embedding it in a resource. 
+
+## Cross namespace resources
+
+Namespace boundaries in Kubernetes provide a barrier for access scopes. Apps or users can be limited to only access resources in a certain namespace. cert-manager is a controller that operates on cluster wide resources, while it may seems interesting to allow access to copy or write certificate data from one namespace to the other it will cause a bypass of this security model for all our users which is not intended and a security issue.
+If this behavior is intended for your use case there are other Kubernetes controllers that will do this for you however we want to suggest a certain caution when installing these. 
+
+## Sign certificates using the Kubernetes CA (used for your nodes)
+
+
+Kubernetes has a Certificate Signing Requests API, and a `kubectl certificates` command which allows you to approve certificate signing requests and have them signed by the certificate authority (CA) of the Kubernetes cluster.
```

Comment:
Have a look at the v1 API.  The trust for kubernetes clusters is actually split into three different signers as opposed to a "the certificate authority of the kubernetes cluster".  There isn't a single authority and CA bundles are not provided in SA token secrets.

Reply:
Need to dig deeper myself into this, this comes from our FAQ section. Proably needs a rewrite